frontend/src/lib/components/PropertyGroupFilters/PropertyGroupFilters.tsx

```diff
@@ -43,36 +44,29 @@ export function PropertyGroupFilters({
         duplicateFilterGroup,
     } = useActions(propertyGroupFilterLogic(logicProps))
 
-    // Update the logic's internal filters when the props change
-    useEffect(() => {
-        setFilters(propertyFilters ?? { type: FilterLogicalOperator.And, values: [] })
-    }, [propertyFilters])
```

Comment:
I'm not a fan of these useEffects, but I think it's needed. Otherwise the component will never update if its value prop (`propertyFilters`) changes.